lib/payment/response_classes/merchant_stripe.rb

```diff
@@ -188,7 +188,8 @@ def <=>(other)
     end
 
     attr_accessor :charge_id, :status, :amount_cents, :currency,
-                  :customer_name, :credit_card_brand, :created_at
+                  :customer_name, :customer_email, :credit_card_brand,
```

Comment:
This shouldn't be added here. The payment service returns the customer_email only as a detail. And we don't use this class in that case.